app/boards/shields/tidbit/tidbit.yaml

```diff
@@ -0,0 +1,7 @@
+identifier: tidbit
```

Comment:
This doesn't seem to account for the `19key` variant?

Technically variants can (should?) be extrapolated from the filesystem as they're `${SHIELD}-${VARIANT}.overlay`.

How do variants differ from "components" described elsewhere in this issue?  They follow the same file system naming conventions.  Should we treat them as one or differently in the metadata?